src/SAWScript/Crucible/LLVM/Override.hs

```diff
@@ -1631,17 +1688,38 @@ executeAllocation opts sc cc (var, LLVMAllocSpec mut memTy alignment sz loc fres
      mem <- readGlobal memVar
      sz' <- instantiateExtResolveSAWSymBV sc cc Crucible.PtrWidth sz
      let l = show (W4.plSourceLoc loc) ++ " (Poststate)"
-     (ptr, mem') <- liftIO $
-       Crucible.doMalloc sym Crucible.HeapAlloc mut l mem sz' alignment
+     (ptr, mem') <- liftIO $ doAllocSymInit sym mem mut alignment sz' l symInit
      writeGlobal memVar mem'
      assignVar cc loc var ptr
 
+doAllocSymInit ::
+  (Crucible.IsSymInterface sym, Crucible.HasPtrWidth wptr, Crucible.HasLLVMAnn sym) =>
+  sym ->
+  Crucible.MemImpl sym ->
+  Crucible.Mutability ->
+  Crucible.Alignment ->
+  W4.SymBV sym wptr {- ^ allocation size -} ->
+  String {- ^ source location for use in error messages -} ->
+  Bool {- ^ source location for use in error messages -} ->
```

Comment:
I think this comment was accidentally copy-pasted from the argument above it, as "source location" doesn't seem to describe what this `Bool` is doing accurately.